modules/superset/tests/superset_with_ingress/main.tf

```diff
@@ -0,0 +1,34 @@
+module "test" {
+  source = "../../"
+
+  name               = "superset"
+  namespace          = "superset"
+  cluster_name       = "buycycle-cluster"
+  supeset_secret_key = "sdgiadgiagdiagdi"
+
+  postgress_config = {
+    create = false
+    host   = "postgress-aws-hostname"
+    port   = "5432"
+    user   = "superset"
+    pass   = "superset"
+    name   = "superset"
+  }
+
+  redis_config = {
+    create = false,
+    host   = "external-redis-hostname",
+    port   = 6379
+  }
+
+  alarms = {
+    enabled   = false
+    sns_topic = ""
+  }
+
+  ingress_config = {
```

Comment:
Do we want to have ingress_confg? can we just have host as primary variable (on root level)?

Reply:
Yes, we need to send the certificate for the host and provide minimum parameters for the ingress; otherwise, the ingress won't be created or connected to the ALB."